Docs for schema admin role #425 (#426)
enonic/doc-xp: docs/iam.adoc

A special role that both authenticated users, and visitors (Anonymous user) all have in common.
The role is for instance used to grant read permissions to publicly available content projects.

:imagesdir: images
image:xp-7110.svg[XP 7.11.0,opts=inline]

> *Schemas Administrator*

Reviewer: ```suggestion
*Schema Administrator*
```

> Grants full access to the Model Studio, including create/delete for virtual apps and create/update/delete for schemas.

Reviewer: ```suggestion
Grants permissions to manipulate virtual applications and schemas.
```
